js/workersrc.js

```diff
@@ -15,3 +15,7 @@ redirectIfNotDisplayedInFrame();
 PDFJS.openExternalLinksInNewWindow = true;
 PDFJS.isEvalSupported = false;
 PDFJS.workerSrc = document.getElementsByTagName('head')[0].getAttribute('data-workersrc');
+
+PDFViewerApplication.download = function() {
+	window.open(decodeURIComponent(window.location.search.substr(6)), '_blank');
```

Comment:
Any chance that you have the debug mode enabled? If so: Disable that.

Thomas has disabled the Content-Disposition with type of attachment when the debug mode is enabled because he wanted to use the SabreDAV browser… 🙈